RK3566 port to 6.17

sydarn opened the pull request:
needs to be rebased, currently breaking other devices.

Seems to work fine on Arc and RGB30.

sydarn committed:
RK3566 port to 6.17

sydarn commented:
Needs a build test on rk3326

porschemad911 commented:
> Needs a build test on rk3326

I'll run a RK3326 test build in the morning for my OGA.

sydarn commented:
> I'll run a RK3326 test build in the morning for my OGA.

No need, it built fine here on my end. Even tested on rg351v, even though that may not have been necessary.

Starting a build for s922x just in case, but I think this is good

sydarn commented:
S922X seems fine.

r3claimer commented:
How's this going? I can move rk3399 to 6.17(or .18) once this is merged.

sydarn commented:
you're right, let me rebase so we get this merged. I was experimenting with other things.

sydarn committed:
rg-arc: swap touch screen axes in dts

This is probably a hack, and I am somehow compensating for it being
rotated elsewhere. But until that is figured out this gets the job done.
